lib/ansible/config/ansible_builtin_runtime.yml

```
@@ -8313,7 +8313,7 @@ plugin_routing:
     ce:
       redirect: community.network.ce
     cloudscale:
-      redirect: community.general.cloudscale
+      redirect: cloudscale_ch.cloud.doc_fragments
```

This should be name of the file under `plugins/doc_fragment/`
```suggestion
      redirect: cloudscale_ch.cloud.api_parameters
```

Simialr for module_utils & inventory

---

Fixed, AFAICS the path for the inventory plugin was already correct.